src/appleseed.shaders/src/appleseed/as_standard_surface.osl

```diff
@@ -298,7 +312,7 @@ shader as_standard_surface
         float max = 1.0,
         string label = "Coating Reflectivity",
         string page = "Coating",
-        string help = "Coating specular amount."
+        string help = "Coating specular reflectivity."
```

Comment:
A very minor thing, but some help strings end with a dot, some others not.

Reply:
You're right, now it's as good a time to fix this as any other. I'll update the remaining nodes affected by this as well.